react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+
+export default function Stopwatch() {
+  const [play, setPlay] = useState(false);
+  const [time, setTime] = useState(0);
+  const [intervalID, setIntervalID] = useState(0);
+
+  function handlePlay() {
+    clearInterval(intervalID);
```

Comment:
Once you've cleared this interval, `intervalID` is no longer valid. It would be best to unset the value at this point.

If you do this, you will then be able to use `intervalID` to tell if you are playing and eliminate the `play` state variable.